index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+<header>
+    <nav class="nav-bar" >
+        <div class="logo"><span>John Doe </span> </div>
+        <div class="menu"><span><img src="/img/Union.png" alt="menu"> </span> </div>
+    </nav>
+</header>
+<main class="main-container">
+    <section class="container">
+
+    
+   <div class="intro">
+
+ 
+    <h1>
+        Hey There. <br>
+I’m Alex
+    </h1>
+    <h3>
+        I’am a Software Developer
+    </h3>
+
+
+
+
+    <div class="intro-details">
+    <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+     
+           
+      
+    </p>   
+    <img src="/img/Social-Media.png" alt="Social-Media"> 
```

Comment:
- [ ] You are using an image for the social icons, instead use a list of icons with a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)